specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/preview/2022-12-15-preview/storageContainers.json

```diff
@@ -0,0 +1,443 @@
+          }
+        }
+      },
+      "delete": {
+        "operationId": "StorageContainers_Delete",
+        "description": "The operation to delete a storage container.",
+        "x-ms-examples": {
+          "DeleteStorageContainer": {
+            "$ref": "./examples/DeleteStorageContainer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/StorageContainerNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No content"
+          }
+        }
+      },
+      "patch": {
+        "operationId": "StorageContainers_Update",
+        "description": "The operation to update a storage container.",
+        "x-ms-examples": {
+          "UpdateStorageContainer": {
+            "$ref": "./examples/UpdateStorageContainer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/StorageContainerNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "storageContainers",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StorageContainersUpdateRequest"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StorageContainers"
+            }
+          },
+          "201": {
+            "description": "Created",
```

Comment:
Patch should not return 201 created

Reply:
will fix in next release. Work item created.